packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -1,15 +1,15 @@
-import DotGovIcon from '../../images/usa-banner-dot-gov.svg';
 import EnglishTranslations from '../../locale/en.json';
-import HttpsIcon from '../../images/usa-banner-https.svg';
-import LockIcon from '../../images/usa-banner-lock.svg';
+import IconDotGov from '../../images/usa-banner-dot-gov.svg';
```

Comment:
why was this renamed?

Reply:
Want to group the svg icons together.

Reply:
Would it be better to keep it's original naming?

Reply:
That makes sense to be prefixing the imports with "Icon" to keep them together